docs/team_sync.md

```diff
@@ -0,0 +1,57 @@
+Workspace Sync
+==============
+
+The term Team is synonymously used for a Slack Workspace.
+
+The configuration option `team_sync` allows to replicate channels and users from Slack to Matrix before a user writes a message.
+
+To have a consistent experience across both sides of the bridge, it is recommended to have this feature enabled.
+
+TODO: Why would you not have this enabled?
```

Comment:
Not sure, but let's leave this comment out of the markdown for now. We can open an issue for this if needed.

Reply:
removed in 51186b6